swift/StableDiffusion/pipeline/StableDiffusionXLPipeline.swift

```diff
@@ -49,19 +52,21 @@ public struct StableDiffusionXLPipeline: StableDiffusionPipelineProtocol {
     ///   - reduceMemory: Option to enable reduced memory mode
     /// - Returns: Pipeline ready for image generation
     public init(
-        textEncoder: TextEncoderXLModel,
+        textEncoder: TextEncoderXLModel?,
         textEncoder2: TextEncoderXLModel,
         unet: Unet,
+        unetRefiner: Unet?,
         decoder: Decoder,
         encoder: Encoder?,
         reduceMemory: Bool = false
     ) {
         self.textEncoder = textEncoder
         self.textEncoder2 = textEncoder2
         self.unet = unet
+        self.unetRefiner = unetRefiner
         self.decoder = decoder
         self.encoder = encoder
-        self.reduceMemory = reduceMemory
+        self.reduceMemory = true
```

Comment:
Was this intentional? I think it should be controlled externally.

Reply:
@ZachNagengast  let's roll this hardcoding back 🙏